src/commands/create.ts

```diff
@@ -57,16 +56,13 @@ const create = async (options: CreateOptions): Promise<void> => {
   }
 };
 
-export const createCommand = (program: Command): void => {
-  program
-    .command("create")
-    .description("Create a new universal contract project.")
-    .option("--no-cache", "Bypass cached repository and re-clone")
-    .option("--verbose", "Enable verbose logging")
-    .option("--output <directory>", "Specify custom output directory or name")
-    .option(
-      "--example <exampleName>",
-      "Specify the example to use and skip the prompt",
-    )
-    .action(create);
-};
+export const createCommand = new Command("create")
```

Comment:
Is this the one we renamed to `new`?

Reply:
That's the one. I've refactored it a bit, so it can be mounted the same way as localnet: https://github.com/zeta-chain/cli/pull/8/commits/a395965c937f5f8f050e2c1a444dc1d7dd1f6072